isis/src/tgo/apps/tgocassisrdrgen/tsts/default/Makefile

```diff
@@ -6,17 +13,14 @@ commands:
 	$(APPNAME) from=$(INPUT)/CAS-MCO-2016-11-22T15.45.50.984-BLU-03000-B1.cub \
 	           to=$(OUTPUT)/CAS-MCO-2016-11-22T15.45.50.984-BLU-03000-B1.img \
 		   > /dev/null;
-
-	# Reingest the PDS4 data
-	tgocassis2isis from=$(OUTPUT)/CAS-MCO-2016-11-22T15.45.50.984-BLU-03000-B1.xml \
-	         to=$(OUTPUT)/CAS-MCO-2016-11-22T15.45.50.984-BLU-03000-B1.cub \
-		 > /dev/null;
-
-	# Compare the output cube from raw2isis to the original input cube to
-	# ensure that they are identical
-	cubediff from=$(INPUT)/CAS-MCO-2016-11-22T15.45.50.984-BLU-03000-B1.cub \
-	         from2=$(OUTPUT)/CAS-MCO-2016-11-22T15.45.50.984-BLU-03000-B1.cub \
-	         to=$(OUTPUT)/cubediffresults.txt \
+			 
+	# Reingest output and make sure the cubes are the same
+	raw2isis from=$(OUTPUT)/CAS-MCO-2016-11-22T15.45.50.984-BLU-03000-B1.img \
```

Comment:
Why switch from `tgocassis2isis` to `raw2isis`? And why did you take out the cubediff?

Reply:
Originally, it was actually using raw2isis, but I changed it to tgocassis2isis with the previous pull request. However, Jeanie said that we should only be testing rdrgen in its apptest, so I changed it back. I also took out the cubediff because I copied over the input cube into the truth folder, so it should automatically do a cubediff when testing the output.

Reply:
cool, thanks for explaining! I should probably have caught that second one, perfect!

Reply:
:+1: